.pre-commit-config.yaml

```diff
@@ -50,7 +50,7 @@ repos:
         pass_filenames: false
       - id: mypy
         name: mypy
-        entry: pdm run python -m mypy
+        entry: mypy --python-executable $(pdm info --python)
```

Comment:
Run standalone mypy seems to be more tricky. If it could list as lint dependencies in pyproject.toml and be locked by pdm because it always needs the running python.😐

Reply:
Look over the changes, the pdm lock file is more clean anyway.